packages/codecs/src/anchor.ts

```diff
@@ -66,6 +67,7 @@ export const AnchorServiceAnchored = strict(
     cid: cidAsString,
     message: string,
     anchorCommit: cidAsString,
+    carBytes: optional(uint8array), // TODO(CDB-2519): make this required once all CAS envs are returning car files
```

Comment:
We should decide what the right name for this field is as this is the contract between ceramic and CAS.  Maybe it should just be `car` instead of `carBytes`?  That or something like `witnessCar`?

Reply:
witnessCar makes sense to me!

Reply:
+1 to just `car`.

Reply:
changed to `witnessCar`